indices/tutorials.js

```diff
@@ -6,17 +6,16 @@ module.exports = new AlgoliaIndex('tutorials', getRecords())
 
 function getRecords () {
   return chain(Object.values(require('electron-i18n').docs['en-US']))
-    .filter(tutorial => !tutorial.isApiDoc && !tutorial.isApiStructureDoc)
+    .filter(tutorial => {
+      const { isApiDoc, isApiStructureDoc, slug } = tutorial
+      return !isApiDoc && !isApiStructureDoc && slug !== 'README'
+    })
     .map(tutorial => {
       const { title, githubUrl, slug, sections, href } = tutorial
       const objectID = `tutorial-${slug}`
       const html = sections.map(section => section.html).join('\n\n')
       const body = cheerio.load(html).text()
 
-      // ignore files that have been renamed
-      if (!title && body.startsWith('Moved to')) return
```

Comment:
Standard 16 doesn't allow `.map` functions to return without a value. These lines of code should be moved to the `filter` function above.